app/component/map/StopsNearYouMap.js

```diff
@@ -166,7 +163,7 @@ function StopsNearYouMap(
     isFetching: false,
     stop: null,
   });
-  const prevMode = useRef();
+  const prevPlace = useRef();
```

Comment:
Now place change seems to update vehicles on map, but it you switch tab e.g. in Tampere journey planner from bus to tram, new vehicles will not show up.